src/hl/plist.rs

```diff
@@ -202,6 +203,20 @@ impl PropertyList {
     }
 }
 
+pub fn set_vlen_manager(plist: crate::internal_prelude::hid_t) -> Result<()> {
+    use std::panic;
```

Comment:
Can this import be moved all the way to the top, with other std imports? (to match the style)